code/clue.js

```diff
@@ -51,9 +212,110 @@ const pickKiller = () => {
+  } else if (guess === '') {
+    document.getElementById('mystery').innerHTML = 'Please make a guess!'
+  } else if (guess === mystery.killer) {
+    document.getElementById('mystery').innerHTML = `Correct! It was ${mystery.killer.firstName} ${mystery.killer.lastName} in the
+${mystery.room} with the ${mystery.weapon.name}.`
+  } else if (guess !== mystery.killer) {
+    document.getElementById('mystery').innerHTML = `Wrong! It was ${mystery.killer.firstName} ${mystery.killer.lastName} in the
+${mystery.room} with the ${mystery.weapon.name}.`
+  }
+}
+
+// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+document.getElementById('reveal').onclick = reveal
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
 // 'The murder was committed by Jacob Green, in the living room with a rope.'
+
+let pickedSuspect
+
+const picked = (name, name2) => {
+  if (guess === '') {
+    document.getElementById(name).style.opacity = "0";
+    document.getElementById(name).style.position = "absolute";
+
+    pickedSuspect = name;
+    guess = name2
+
+    document.getElementById('suspectImage').src = guess.image
+    document.getElementById('suspectImage').style.opacity = 1;
+  } else {
+    // document.getElementById(pickedSuspect).style.opacity = "1";
+    // document.getElementById(pickedSuspect).style.position = "static";
+    // document.getElementById(name).style.zIndex = "0"
+
+    // pickedSuspect = name;
+    // guess = name2;
+    // document.getElementById('suspectImage').src = guess.image
+    // document.getElementById(name).style.opacity = "0"
+    // document.getElementById(name).style.position = "absolute";
+    // document.getElementById(name).style.zIndex = "-1"
+
+  }
+}
+
+const released = () => {
+  if (guess.firstName !== undefined) {
+    document.getElementById('suspectImage').style.opacity = 0;
+    guess = ''
+
+
+  }
+  if (pickedSuspect === 'mrsWhite') {
+    document.getElementById('mrsWhite').style.opacity = "1"
+    document.getElementById('mrsWhite').style.position = "static"
+  } else if (pickedSuspect === 'mrGreen') {
+    document.getElementById('mrGreen').style.opacity = "1"
+    document.getElementById('mrGreen').style.position = "static"
+  } else if (pickedSuspect === 'profPlum') {
+    document.getElementById('profPlum').style.opacity = "1"
+    document.getElementById('profPlum').style.position = "static"
+  } else if (pickedSuspect === 'colMustard') {
+    document.getElementById('colMustard').style.opacity = "1"
+    document.getElementById('colMustard').style.position = "static"
+  } else if (pickedSuspect === 'missScarlet') {
+    document.getElementById('missScarlet').style.opacity = "1"
+    document.getElementById('missScarlet').style.position = "static"
+  } else if (pickedSuspect === 'mrsPeacock') {
+    document.getElementById('mrsPeacock').style.opacity = "1"
+    document.getElementById('mrsPeacock').style.position = "static"
+  }
```

Comment:
Is there any way to reduce the amount of repeated strings ('mrsPeacock') here?  

What would happen if you did
```Javascript
    document.getElementById(pickedSuspect).style.opacity = "1"
    document.getElementById(pickedSuspect).style.position = "static"
```